creator-node/compose/docker-compose.yml

```diff
@@ -28,6 +28,7 @@ services:
       - creatorNodeEndpoint=${creatorNodeEndpoint}
       - snapbackDevModeEnabled=${snapbackDevModeEnabled}
       - spOwnerWallet=${spOwnerWallet}
+      - isUserMetadataNode=${CREATOR_NODE_IS_USER_METADATA}
```

Comment:
nit this casing is inconsistent

Reply:
yeah it is but it follows established pattern